packages/utilities/psammead-locales/moment/index.stories.jsx

```diff
@@ -50,6 +52,8 @@ const locales = [
   { name: 'Ukrainian', locale: 'uk' },
   { name: 'Yoruba', locale: 'yo' },
   { name: 'Kyrgyz', locale: 'ky' },
+  { name: 'Afaan Oromoo', locale: 'om' },
+  { name: 'Amharic', locale: 'am' },
```

Comment:
LGTM, but could you sort the locales he by name?

Reply:
Sorted.